rfcs/rfc-005-branches-tags-and-channels.md

```diff
@@ -0,0 +1,52 @@
+```
+========================= master branch (edge channel) =======================>
+         \                      \                     \
+          \___v0.7.0 tag         \                     \
+           \                      \         v0.9.0 tag__\
+            \          v0.8.0 tag__\                     \
+ v0.7.1 tag__\                      \                 v0.9 branch (beta channel)
+              \___v0.7.2 tag         \___v0.8.1 tag
+               \                      \
+                \                      \
+           v0.7 branch         v0.8 branch (stable channel)
+
+```
+
+## Branches and Tags
+
+### master branch
+All new development occurs on the `master` branch.
+
```

Comment:
A very brief description of the problem solved by "upstream first" may help with on-boarding. The example I usually use is the regression that comes from a release that does not include a patch in a previous release because it was deemed a dirty hack that couldn't be upstreamed to master.

Reply:
✔️